azure-pipelines.yml

```diff
@@ -96,6 +96,22 @@ jobs:
   - bash: ci/scripts/check-vendoring.sh
     displayName: Check vendored directories are up-to-date
 
+- job: verible_format
+  displayName: Run code formatting checks (using an allow list)
+  pool:
+    vmImage: ubuntu-18.04
+  steps:
+  - template: ci/install-package-dependencies.yml
+  - bash: |
+    util/verible-format.sh
+    if [ $? != 0 ]; then
+      echo "Verilog format with Verible failed. Run 'util/verible-format.sh' to check and fix all errors."
+      echo "This flow is currently experimental and failures can be ignored."
+      exit 1
+    fi
```

Comment:
This block should be indented for the syntax error to go away.

Reply:
Moved the code to a CI script.